core/modules/field_ui/field_ui.admin.inc

```diff
@@ -2085,6 +2085,20 @@ function field_ui_field_edit_form_validate($form, &$form_state) {
     if (function_exists($function)) {
       $function(NULL, NULL, $field, $instance, LANGUAGE_NONE, $items, $errors);
     }
+    // Checking for a difference between the existing default values
+    // and the cardinality setting.
+    $non_empty_items_count = 0;
+    foreach ($items as $item) {
+      if (!empty($item['value'])) {
+        $non_empty_items_count++;
+      }
+    }
+    if ($cardinality === 'number' && $non_empty_items_count > $cardinality_number) {
+      $errors[$field['field_name']][LANGUAGE_NONE][0][] = array(
+        'error' => 'cardinality_smaller_than_existing_default_values',
+        'message' => t('%name: you have entered more default values (@count_non_empty_items) than allowed by the %cardinality_setting_label setting.', array('%name' => $instance['label'], '@count_non_empty_items' => $non_empty_items_count, '@cardinality' => $cardinality_number, '%cardinality_setting_label' => 'Number of values')),
```

Comment:
The array for the string has one more item than is in the string: '@cardinality' => $cardinality_number,

Reply:
Also, `'%cardinality_setting_label' => 'Number of values'` does not need to be escaped and can just be placed in the string.
Also, also, the array probably should be broken out into multiple lines, but that's a Drupal coding standard, and I don't know it is the same with Backdrop.